cloudflare-config.toml

```diff
@@ -11,6 +11,12 @@ title = "TPW" # Teczki Polskiego Wrestlingu. In English, Tales of Polish Wrestli
 
 theme = "mensch"
 
+taxonomies = [
+  { name = "org", render = false },
```

Comment:
A better name is `chrono`, as it will be used to mark which chronologies events belong to.